demo/App.jsx

```diff
@@ -5,7 +5,9 @@ import {WebexMeetingWidget} from '../src';
 import './App.css';
 
 export default function App() {
-  return <WebexMeetingWidget />;
+  return (
+    <WebexMeetingWidget accessToken="<YOUR_ACCESS_TOKEN>" meetingDestination="MEETING_DESTINATION"></WebexMeetingWidget>
```

Comment:
Since it has no children
```suggestion
    <WebexMeetingWidget accessToken="<YOUR_ACCESS_TOKEN>" meetingDestination="MEETING_DESTINATION" />
```